### ISSUE TITLE
Upgrade to ACMEv2

### ISSUE THREAD

**Issue by radimklaska** [morpht/letsencrypt_drupal#6]
> According to our records, the software client you're using to get Let's
> Encrypt TLS/SSL certificates issued or renewed at least one HTTPS certificate
> in the past two weeks using the ACMEv1 protocol. Here are the details of one
> recent ACMEv1 request from each of your account(s):

The underlying issue is that we are currently stuck on an old version of `dehydrated`
```
    # Workaround!
    #
    # @ToDo: Update to new version ASAP!
    #
    # New version requests all challenges for all domains at once.
    #
    # Our script builds on old approach where
    # the whole challenge and verification process
    # is proccesed site by site.
    #
    # Using this version from Dec 2017 works.
    # But still includes this bug: https://github.com/dehydrated-io/dehydrated/issues/450
```
^ from https://github.com/morpht/letsencrypt_drupal/blob/master/letsencrypt_drupal.sh#L81

**Comment by radimklaska** [comment]
`Account creation on ACMEv1 is disabled. Please upgrade your ACME client to a version that supports ACMEv2` Time is up!

**Comment by radimklaska** [comment]
Fixed in 7059684
